### ISSUE TITLE
Make XHR FormData work with this encoding (was: JSONFormData object)

### ISSUE THREAD

**Issue by roman01la** [darobin/formic#18]
There's always a thing about fetching form data to construct JSON object. We need an analogue to FormData object.

**Comment by darobin** [comment]
I don't believe that we need a specific new `JSONFormData`, but we do need to make sure that XHR does not override the enctype to be multipart/form-data when FormData is built from a form.

**Comment by Nijso** [comment]
I no longer submit, I claim all that was and that is,
